OpenTabletDriver/Configurations/XP-Pen/Deco mini 4.json

```
@@ -0,0 +1,39 @@
+{
+  "Name": "XP-Pen Deco Mini 4",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 101.6,
+      "Height": 76.2,
+      "MaxX": 20320.0,
+      "MaxY": 15240.0
+    },
+    "Pen": {
+      "MaxPressure": 2048,
+      "Buttons": {
+        "ButtonCount": 2
+      }
+    },
+    "AuxiliaryButtons": {
+      "ButtonCount": 6
+    },
+    "MouseButtons": null,
+    "Touch": null
+  },
+  "DigitizerIdentifiers": [
+    {
+      "VendorID": 10429,
+      "ProductID": 2345,
+      "InputReportLength": 12,
+      "OutputReportLength": 12,
+      "ReportParser": "OpenTabletDriver.Vendors.XP_Pen.XP_PenReportParser",
+      "FeatureInitReport": null,
+      "OutputInitReport": [
+        "ArAEAAAAAAAAAAAA"

```

No need to specify trailing zeroes.

```suggestion
        "ArAE"

```

---

Oh okay, sorry.